l10n_es_aeat_sii_oca/wizards/account_move_reversal.py

```diff
@@ -9,8 +9,12 @@ class AccountMoveReversal(models.TransientModel):
     _inherit = "account.move.reversal"
 
     def _default_sii_refund_type_required(self):
-        invoices = self.env["account.move"].browse(self.env.context.get("active_ids"),)
-        return any(invoices.mapped("company_id.sii_enabled"))
+        invoices = self.env["account.move"].browse(self.env.context.get("active_ids"))
+        # If any of the invoices part of the active_ids match the criteria,
+        # show the field
+        return any(invoices.mapped("company_id.sii_enabled")) and invoices.filtered(
+            lambda i: i.move_type in ("in_invoice", "out_invoice")
+        )
```

Comment:
Please simplify the expression and in v13, it's `type` AFAIK:
```suggestion
        return bool(invoices.filtered(
            lambda i: i.type in ("in_invoice", "out_invoice") and i.company_id.sii_enabled
        )
```

Reply:
> in v13, it's type AFAIK

Sorry, MB.

Changes made :+1: